packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -301,24 +302,59 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
-    valueLabel: [
-      classNames.valueLabel,
+    regularLabel: [
+      classNames.regularLabel,
+      getHighContrastSelector(disabled ? 'GrayText' : 'WindowText'),
       {
-        flexShrink: 1,
-        width: 30,
-        lineHeight: '1' // using a string here meaning it's relative to the size of the font
+        position: 'absolute',
+        height: '0px',
+        background: theme.palette.neutralTertiaryAlt,
+        transform: 'translateY(10px) translateX(-6px)'
+      },
+      vertical && {
+        margin: '0 auto 20px',
+        transform: 'translate(10px,10px) '
+      },
+      disabled && {
+        background: theme.palette.neutralLight
+      }
+    ],
+
+    regularTick: [
+      classNames.regularTick,
+      getHighContrastSelector(disabled ? 'GrayText' : 'WindowText'),
+      {
+        position: 'absolute',
+        background: '#fff'
+      },
+      disabled! && {
+        background: theme.palette.neutralLight
       },
       vertical
         ? {
-            margin: '0 auto',
-            whiteSpace: 'nowrap',
-            width: 40
+            width: '16px',
+            height: '1px',
+            transform: getRTL() ? 'translateX(6px)' : 'translateX(-6px)'
           }
         : {
-            margin: '0 8px',
-            whiteSpace: 'nowrap',
-            width: 40
+            width: '2px',
```

Comment:
Do the same pattern as you did for `regularLabel` section. Set the default horizontal styles on the main style object of the `regularTick` and then use the `vertical && {...}` later. No need of this ternary here.